static/js/compose_ui.js

```diff
@@ -8,6 +8,17 @@ exports.autosize_textarea = function (textarea) {
     // Since this supports both compose and file upload, one must pass
     // in the text area to autosize.
     autosize.update(textarea);
+    $("#compose-textarea").css(
+        "max-height",
+        parseInt($("#bottom_whitespace").outerHeight(), 10) -
```

Comment:
Hey @timabbott , Any suggestions on fixing the failing pipeline (been looking into it for a while now) 
More specifically , When tested by <code>./tools/lint --groups=frontend --skip=gitlint</code> throws an error as 
<pre>
Error: You must create a stub for $("#bottom_whitespace").outerHeight
Sep 22 17:19:24     at Object.get (/__w/zulip/zulip/frontend_tests/zjsunit/zjquery.js:43:297)
Sep 22 17:19:24     at Object.exports.autosize_textarea (/__w/zulip/zulip/static/js/compose_ui.js:4:134)
Sep 22 17:19:24     at Object.exports.handle_keydown (/__w/zulip/zulip/static/js/compose.js:67:624)
Sep 22 17:19:24     at all_markdown_test (/__w/zulip/zulip/frontend_tests/node_tests/compose.js:25:171)
Sep 22 17:19:24     at /__w/zulip/zulip/frontend_tests/node_tests/compose.js:44:27
Sep 22 17:19:24     at exports.with_overrides (/__w/zulip/zulip/frontend_tests/zjsunit/namespace.js:13:1238)
Sep 22 17:19:24     at global.run_test (/__w/zulip/zulip/frontend_tests/zjsunit/index.js:11:630)
Sep 22 17:19:24 (...)
</pre>